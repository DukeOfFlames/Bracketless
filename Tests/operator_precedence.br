START
#drucke{3 + 4 * 5 ^ 6}
#drucke{5 ^ 6 * 4 + 3}
END