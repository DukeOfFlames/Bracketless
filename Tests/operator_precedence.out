62503
62503
